lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -595,6 +595,37 @@ void RuntimeAssertOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
   });
 }
 
+//===----------------------------------------------------------------------===//
+// Aten__Or__BoolOp
+//===----------------------------------------------------------------------===//
+
+void Aten__Or__BoolOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
```

Comment:
Should write this as a folder, not a canonicalizer.

Reply:
Done